inst/recorder/app.R

```diff
@@ -283,9 +283,10 @@ generateTestCode <- function(events, name, seed, useTimes = FALSE,
     if (load_mode) {
       'app <- ShinyLoadDriver$new()'
     } else {
-
       paste0(
-        'app <- ShinyDriver$new("', paste("..", app$getAppFilename(), sep = "/"), '"',
+        # TODO: test with RMD
+        # TODO: Windows compat with / sep?
+        'app <- ShinyDriver$new("', paste(app$getRelativePathToApp(), app$getAppFilename(), sep=.Platform$file.sep), '"',
```

Comment:
I think I've convinced myself that this is actually right as-is, but let me know if I'm missing something.

`getAppFilename()` just returns the `basename` if in RMD or NULL if a Shiny app. The NULL causes `file.path` to barf, which is why we need to use paste (I added a comment to clarify)

The relativePathToApp is always either `../` or `../../`, so I don't think we'll need to worry about backslash escaping.

I tested a `recordTest()` -> `testApp()` flow on Windows just now on an RMD and it was happy.

Let me know what you think!

Reply:
I think it would be better to use `sep="/"`, since it would simplify things -- no need for `.Platform$file.sep`, since Windows will work with regular slashes.